scripts/dump-users.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const dumpUsers = require('./dump-users/index')
+const fs = require('fs')
+const path = require('path')
+const program = require('commander')
+
+program
+  .option('-e, --emails [emails]', 'Email addresses to dump, comma separated')
+  .option('-u, --uids [uids]', 'User IDs to dump, comma separated')
+  .option('-i, --input <filename>', 'Input filename from which to read input if not specified on the command line')
+  .option('-p, --pretty', 'Display output in a pretty fashion')
+  .parse(process.argv)
+
+if (! program.emails && ! program.uids) {
+  console.error('One of `emails` or `uids` must be specified')
+  process.exit(1)
+} else if (program.emails && program.uids) {
+  console.error('Only one of `emails` or `uids` can be specified, not both')
+  process.exit(1)
+}
+
+let emails = []
+let uids = []
+
+if (program.emails) {
+  emails = getItems('emails')
+} else if (program.uids) {
+  uids = getItems('uids')
+}
+
+if (! emails.length && ! uids.length) {
+  console.error('No `emails` or `uids` specified')
+  process.exit(1)
+}
+
+
+const keys = emails.length ? emails : uids
+const dbFunc = emails.length ? 'emailRecord': 'account'
```

Comment:
I think we should use `accountRecord` here instead of `emailRecord`. I remember trying to depreciate [emailRecord](https://github.com/mozilla/fxa-auth-server/issues/1999#issuecomment-316447124) but ran into some snags and punted on the idea. However, at the least I can put up a PR to add some notes that `emailRecord` is only to be used internally.